functions/package.json

```diff
@@ -3,18 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "tsc": "tsc --project tsconfig.json",
-    "start": "yarn tsc && node dist/iCalFunctions.js"
+    "tsc": "tsc",
+    "build": "yarn tsc && ncc build dist/index.js -o compiled -m",
+    "serve": "yarn tsc && firebase serve --only functions",
+    "shell": "yarn tsc && firebase functions:shell",
+    "start": "yarn shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
   },
+  "main": "dist/index.js",
   "dependencies": {
-    "firebase": "^7.4.0",
```

Comment:
This is useless in functions. It's supposed to be used for frontend. Including it WILL mess up with deployment.